Painting Exhibit tweaks

cacogen opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

### Player
- Persistent paintings change atom name based on their painting (e.g. _painting - Polar Bear on white_, instead of _Public Painting Exhibit_)
- Persistent paintings without canvases are now referred to as mountings for their respective exhibit
- Simplifies painting descriptions and moves wirecutting and persistence explanations to a notice shown on examine
- Slightly different descriptions based on whether it has a painting or not (look at the code to hopefully understand why)
- Removes the OK_ART art element from persistent paintings as it's unclear as a player why it's the same for every painting or whether it can be affected (I'd like to add voting to paintings at some point and base appraisal off of that)

### Code
- Adds a new desc_with_canvas var to the painting structure that's used to set its description when it has a canvas
- Adds a new update_name_and_desc() proc to the painting structure that's called when a canvas is added or removed
- Replaces use of C variable with named variables (e.g. crayon, canvas, current_canvas) in artstuff.dm for readability
- Adds autodoc description for two painting structure vars


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Looks better, is more immersive, makes more sense and is more intuitive. Code is slightly more readable.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: cacogen
tweak: Painting exhibit names and descriptions are different
del: Paintings no longer tell you they're fair to middling
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

cacogen committed:
Painting Exhibit tweaks

- Replaces use of C variable with named variables (e.g. crayon, canvas, current_canvas) in artstuff.dm for readability
- Simplifies painting structure's descriptions and moves wirecutting and persistence tutoring to a notice shown on Examine()
- Adds autodoc for two painting structure vars
- Adds a new desc_with_canvas var to painting structure that's used to set its description when it has a canvas
- Adds a new update_name_and_desc() proc to painting structure called when a canvas is added or removed
- Makes it so painting structures are named based off the painting rather than keeping their generic name

Mickyan commented on code/game/objects/structures/artstuff.dm:
```diff
@@ -261,57 +263,68 @@
-		C.ui_interact(user)
+	if(current_canvas)
+		current_canvas.ui_interact(user)
+	. += "<span class='notice'>Any painting placed here will be archived at the end of the shift. Use wirecutters to remove the painting.</span>"
```
The part about archiving should be checking for a persistence_id, and still display if there's no canvas currently present

cacogen committed:
Changes default names to refer to the "exhibits" as exhibit mountings

- Changes "Secure Painting Exhibit" to "Curated Painting Exhibit" as it's more intuitive

Farquaar commented:
>
>     * Simplifies painting descriptions and moves wirecutting and persistence explanations to a notice shown on examine

THANK YOU

tgstation-server commented:
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.
